### PR TITLE
Add PR testing to Ansible

### PR DESCRIPTION
**Pull request by dLobatog**

### CODE REVIEW

File: puppet/modules/slave/templates/test_pull_requests_foreman_ansible.json.erb [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+    "settings": {
+        "test_settings": {
+            "name": "test",
+            "test_prefix": "Test Results:",
+            "allow_multiple": true,
+            "run_without_trigger": true,
+            "branches": {
+                "*": {
+                    "jenkins_job_name": "test_plugin_pull_request",
+                    "downstream_job_name": "test_plugin_foreman_ansible_master",
```
Comment (domcleal) [inline puppet/modules/slave/templates/test_pull_requests_foreman_ansible.json.erb]: This job testing master doesn't currently exist, you'll need to create it first (step 2 at http://projects.theforeman.org/projects/foreman/wiki/Jenkins#Adding-a-new-Foreman-plugin) or change to `true` to not test master at all.

Comment (domcleal) [inline puppet/modules/slave/templates/test_pull_requests_foreman_ansible.json.erb]: Sorry, I see now that's what the first file does. Could you also update the slave::pr_test_config list in puppet/modules/slave/manifests/init.pp? (Step 2 of the second part.)

**Comment by dLobatog** [comment]
Updated - I don't have permissions to set the hook in the foreman_ansible project, so that's missing

**Comment by domcleal** [comment]
Thanks, it'll be active within the next hour or so.

**Pull request merged**